src/OrchardCore.Modules/OrchardCore.Email/OrchardCore.Email.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+    <PackageReference Include="MailKit" Version="2.1.3" />
```

Comment:
upgrade to latest :)

Reply:
2.1.4

Reply:
Oh, this needs to be moved to the props file

Reply:
Sure